docs/pages/repo/docs/core-concepts/caching.mdx

```diff
@@ -71,7 +71,7 @@ To override the default cache output behavior, pass an array of globs to a [`pip
 }
 ```
 
-If your task does not emit any files (e.g. unit tests with Jest), set `outputs` to an empty array (i.e. `[]`), and Turborepo will cache the logs for you.
+Turborepo automatically records and caches the logs of every task. You do not need to specify anything to get this behavior. If your task does not emit any files (e.g. unit tests with Jest) you can simply omit `outputs`.
```

Comment:
I like the rewording around the logs, but feel like we should keep the continuity from caching file outputs and explain how that relates to caching logs after, rather than jumping directly to logs.

```suggestion
If your task does not emit any files (e.g. unit tests with Jest) you can omit `outputs`. Even without any file outputs, Turborepo automatically records and caches the logs of every task. If no inputs change (i.e. if there is a cache hit), subsequent runs will replay these logs.
```